src/app/modules/group/pages/group-access/group-access.component.ts

```diff
@@ -0,0 +1,38 @@
+import { Component, Input, OnChanges } from '@angular/core';
+import { Group } from '../../http-services/get-group-by-id.service';
+import { GetItemByIdService } from '../../../item/http-services/get-item-by-id.service';
+import { ReplaySubject, of } from 'rxjs';
+import { switchMap } from 'rxjs/operators';
+import { mapToFetchState } from '../../../../shared/operators/state';
+
+@Component({
+  selector: 'alg-group-access',
+  templateUrl: './group-access.component.html',
+  styleUrls: [ './group-access.component.scss' ],
+})
+export class GroupAccessComponent implements OnChanges {
+  @Input() group?: Group;
+
+  private readonly rootActivityId$ = new ReplaySubject<string | null>(1);
+
+  itemState$ = this.rootActivityId$.pipe(
+    switchMap(rootActivityId => {
+      if (!rootActivityId) {
+        return of(null);
+      }
+      return this.getItemByIdService.get(rootActivityId).pipe(
+        mapToFetchState()
+      );
+    })
+  );
```

Comment:
What confused me in my previous comment was that I would have expected:
```suggestion
  itemState$ = this.rootActivityId$.pipe(
    switchMap(rootActivityId => {
      if (!rootActivityId) {
        return of(null);
      }
      return this.getItemByIdService.get(rootActivityId);
    }),
    mapToFetchState()
  );
```
Your solution results in a `Observable<FetchState<Item>|null>` (either "null" or a ready/fetching/error state) while I expected a `Observable<FetchState<Item|null>>`, a state which is either error, fetching, or ready of either nothing (if no root activity for this group) or an item. 
For now, that's quite okay, but when we'll add more fetching combined with this one, probably that will not work well anymore.

Reply:
Fixed